test/batch_norm.jl

```diff
@@ -0,0 +1,26 @@
+using Test
+using Bijectors, ForwardDiff, LinearAlgebra
+using Random: seed!
+
+seed!(1)
+
+@testset "InvertibleBatchNorm" begin
+    z = randn(20, 2)
+    flow = Bijectors.InvertibleBatchNorm(2)
+    flow.active = false
+    @test inv(inv(flow)) == flow 
+    @test inv(flow)(flow(z)) ≈ z
+    @test (inv(flow) ∘ flow)(z) ≈ z
+
+    @test_throws AssertionError forward(flow, randn(2,10))
+    
+    @test logabsdetjac(inv(flow), flow(z)) ≈ - logabsdetjac(flow, z)
+
+    y = flow(z)
+    @test log(abs(det(ForwardDiff.jacobian(flow, z)))) ≈
+     sum(logabsdetjac(flow, z)) rtol=1e-4 # fails return double
```

Comment:
Making `Float32` optional in the constructor could fix this.

Reply:
Thanks! That helped.